records2test/DC-1047/sm1.edit

```diff
@@ -0,0 +1,42 @@
+! PUBLICATION PROFORMA                   Version 48:  21 May 2018
+!
+! P22.  FlyBase reference ID (FBrf) or "new"  *U :FBrf0075340
+! P2.   Parent multipub abbreviation               *w :
+! P40.  Flag Cambridge for curation           CAMCUR :
+! P41.  Flag Harvard for curation            HARVCUR :
+! P42.  Flag Ontologists for curation           ONTO :
+! P43.  Flag Disease for curation            DISEASE :
+! P44.  Disease(s) relevant to FBrf      [free text] :
+! P19.  Internal notes                   *H :Record to test additional checking of FBco symbol when F1f = new
```

Comment:
Actually this is testing check for incorrect F11 data for FBcos?

Reply:
thanks - have updated comment so will get fixed next time I do a pull request